data/search.go

```diff
@@ -131,7 +131,7 @@ func (r *SearchRepository) GetSearchSummaryByUniverse(searchText string, univers
 	LEFT JOIN data_list_measurements ON data_list_measurements.measurement_id = measurement_series.measurement_id
 	LEFT JOIN categories ON categories.data_list_id = data_list_measurements.data_list_id
 	LEFT JOIN feature_toggles ON feature_toggles.universe = series.universe AND feature_toggles.name = 'filter_by_quarantine'
-	WHERE series.universe = UPPER(?)
+	WHERE series.universe LIKE CONCAT('%',?,'%')
```

Comment:
We'll have to make sure we don't end up with a universe named OCO or any other interior subset of UHEROCOH

Reply:
Well, yes, but this is just a temporary bandaid anyway, right? So I don't think that will be a problem...